configs/sst_cs_apps-ruby.yaml

```diff
@@ -7,6 +7,26 @@ data:
 
   packages:
   - ruby
+  - ruby-default-gems
+  - ruby-devel
+  - ruby-libs
+  - rubygem-bigdecimal
+  - rubygem-bundler
+  - rubygem-io-console
+  - rubygem-irb
+  - rubygem-json
+  - rubygem-minitest
+  - rubygem-power_assert
+  - rubygem-psych
+  - rubygem-rake
+  - rubygem-rbs
+  - rubygem-rdoc
+  - rubygem-rexml
+  - rubygem-rss
+  - rubygem-test-unit
+  - rubygem-typeprof
+  - rubygems
+  - rubygems-devel
```

Comment:
If this should list all the binary RPMs instead of SRPM, then this is correct. But I'd rather leave it somebody else to review.

Reply:
yes it's binary RPMs